app.js

```diff
@@ -1,173 +1,104 @@
 var hours = ['10AM', '11AM', '12AM', '1PM', '2PM', '3PM', '4PM', '5PM',];
 
-var stores = [
-{
-  name: 'Pike Place',
-  min: 17,
-  max: 88,
-  avgSales: 5.2,
-  hourlySales: [],
-  totals: 0,
-  grandTotal: 0,
-  customerPerHour: function () {
+function Store(name, min, max, avgSales) {
+  this.name = name;
+  this.min = min;
+  this.max = max;
+  this.avgSales = avgSales;
+  this.hourlySales = [];
+  this.totals = 0;
+}
+
+Store.prototype.customerPerHour = function(min, max) {
```

Comment:
You don't need parameters in this method, given that you're using the properties of `this` explicitly in your return statement.